`kusto_database ` support data source `kusto_database_data_source.go` (#16180)
hashicorp/terraform-provider-azurerm: website/docs/d/kusto_database.html.markdown

---
subcategory: "Data Explorer"
layout: "azurerm"
page_title: "Azure Resource Manager: azurerm_kusto_database"
description: |-
  Manages Kusto / Data Explorer Database
---

# Data Source: azurerm_kusto_database

Use this data source to access information about an existing Kusto Database

## Example Usage

```hcl
data "azurerm_kusto_database" "example" {
  name                = "my-kusto-database"
  resource_group_name = "test_resource_group"
  cluster_name        = "test_cluster"
}
```

## Argument Reference

The following arguments are supported:

> * `name` - The name of the Kusto Database to create.

Reviewer: ```suggestion
* `name` - (Required) The name of the Kusto Database.
```

> * `resource_group_name` - Specifies the Resource Group where the Kusto Database should exist.

Reviewer: ```suggestion
* `resource_group_name` - (Required) The Resource Group where the Kusto Database exists.
```

> * `cluster_name` - Specifies the name of the Kusto Cluster this database will be added to.

Reviewer: ```suggestion
* `cluster_name` - (Required) The name of the Kusto Cluster this database is added to.
```
